src/encoder/encodermp3.cpp

```diff
@@ -254,23 +254,21 @@ void EncoderMp3::setEncoderSettings(const EncoderSettings& settings)
     int modeoption = settings.getSelectedOption(EncoderMp3Settings::ENCODING_MODE_GROUP);
     m_encoding_mode = (modeoption==0) ? vbr_off : (modeoption==1) ? vbr_abr : vbr_default;
 
-    if (m_encoding_mode != vbr_off) {
-        if (lame_set_VBR_quality) {
-            // vbr range 0 to 9.999 (best to worst). slider range (0 to max) worse to best.
-            int val = settings.getQualityValues().size() - 1 - settings.getQualityIndex();
-            if (val < 10) {
-                m_vbr_index = val;
-            } else {
-                m_vbr_index = 9.999f;
-            }
+    if (m_encoding_mode == vbr_off) {
+        if (m_bitrate > 100 ) {
+            m_stereo_mode = JOINT_STEREO;
         } else {
-            // vbr range 0 to 9 (best to worst). slider range (0 to max) worse to best.
-            int val = settings.getQualityValues().size() - 1 - settings.getQualityIndex();
-            if (val < 10) {
-                m_vbr_index = val;
-            } else {
-                m_vbr_index = 9;
-            }
+            m_stereo_mode = MONO;
+        }
+    } else {
+        // Inverting range: vbr 0 best, 9 worst. slider 0 min to max.
+        int val = settings.getQualityValues().size() - 1 - settings.getQualityIndex();
+        if (val < 8) {
+            m_stereo_mode = JOINT_STEREO;
+            m_vbr_index = val;
+        } else {
+            m_vbr_index = val-4;
```

Comment:
No. Here it is all about bitrate.
The usual range for VBR is from 0 to 6, which go from 260 to 130. (At this range it works at 44Khz. lower than that and starts reducing the sampling rate)
To add lower bitrates, I've used mono encoding, and given that mono needs less bitrate, I've increased the quality. 
So, with this, the bitrate range keeps mostly similar, and the quality is kept not too low thanks to changing to mono.